### PR TITLE
Update ledger overview list

### PR DESCRIPTION
**Pull request by jwhyte-da**
New overview of ledgers from Rodrigo

Screen Shot 2022-02-22 at 12 21 09 PM

CHANGELOG_BEGIN
CHANGELOG_END

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description
- [ ] If you mean to change the status of a component, please make sure you keep [the Component Status page](https://github.com/digital-asset/daml/blob/main/docs/source/support/component-statuses.rst) up to date.

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: docs/source/deploying.rst [hunk 1]
```diff
@@ -1,12 +1,43 @@
 .. Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 .. SPDX-License-Identifier: Apache-2.0
 
-Deploying to Daml Ledgers
-=========================
+Overview of Daml ledgers
```
Comment (cocreature) [inline docs/source/deploying.rst:4]: I thought the plan was that we drop this completely for 2.0 and instead put this on daml.com instead of docs.daml.com?

Comment (jwhyte-da) [reply docs/source/deploying.rst:4]: In a meeting this morning we decided to update the content until we can get a spot in the new digitalasset.com site that's currently being built.

**Review by cocreature — APPROVED** [review]

File: docs/source/deploying.rst [hunk 2]
```diff
@@ -1,12 +1,43 @@
...
 
 .. toctree::
   :maxdepth: 2
+  :hidden:
 
-  deploy/index
   deploy/generic_ledger
-  deploy/ledger-topologies
\ No newline at end of file
+  deploy/ledger_topologies
```
Comment (cocreature) [inline docs/source/deploying.rst:43]: why did you change this?

Comment (jwhyte-da) [reply docs/source/deploying.rst:43]: David is removing in a different PR

Comment (jwhyte-da) [reply docs/source/deploying.rst:43]: David is removing ledger_topologies in a different PR. I removed deploy/index to have less pages now that content is so minimal. I added the table to deploying.rst, the "actual" index that we use for assembly

File: docs/source/daml/reference/packages.rst [hunk 3]
```diff
@@ -20,7 +20,7 @@ You can specify a different path for the Daml archive by using the ``-o`` flag:
 
   daml build -o foo.dar
 
-For details on how to upload a Daml archive to the ledger, see the :ref:`deploy documentation <deploy-ref_overview>`. The rest of this page will focus on how to import a Daml package in other Daml projects.
+The rest of this page will focus on how to import a Daml package in other Daml projects.
```
Comment (cocreature) [inline docs/source/daml/reference/packages.rst:23]: why did you delete that? The reference seems very useful.

Comment (jwhyte-da) [reply docs/source/daml/reference/packages.rst]: Adding back

File: docs/source/daml/reference/packages.rst [hunk 4]
```diff
@@ -20,7 +20,7 @@ You can specify a different path for the Daml archive by using the ``-o`` flag:
 
   daml build -o foo.dar
 
-For details on how to upload a Daml archive to the ledger, see the :ref:`deploy documentation <deploy-ref_overview>`. The rest of this page will focus on how to import a Daml package in other Daml projects.
+For details on how to upload a Daml archive to the ledger, see the :ref:`deploy documentation </deploying>`. The rest of this page will focus on how to import a Daml package in other Daml projects.
```
Comment (jwhyte-da) [inline docs/source/daml/reference/packages.rst:23]: This link is not working, I don't know what the right sphynx syntax is for links. The others work...

File: docs/source/daml/reference/packages.rst [hunk 5]
```diff
@@ -20,7 +20,7 @@ You can specify a different path for the Daml archive by using the ``-o`` flag:
 
   daml build -o foo.dar
 
-For details on how to upload a Daml archive to the ledger, see the :ref:`deploy documentation <deploy-ref_overview>`. The rest of this page will focus on how to import a Daml package in other Daml projects.
+For details on how to upload a Daml archive to the ledger, see the :ref:`deploy documentation <././deploying>`. The rest of this page will focus on how to import a Daml package in other Daml projects.
```
Comment (cocreature) [inline docs/source/daml/reference/packages.rst:23]:
```suggestion
For details on how to upload a Daml archive to the ledger, see the :doc:`deploy documentation </deploying>`. The rest of this page will focus on how to import a Daml package in other Daml projects.
```
:ref: references a label that you set somewhere. You can see the previous one in the deleted file via `.. _deploy-ref_overview:`. `:doc` is used to reference a specific document which you’re doing here.

This is documented in https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html
